### DISCUSSION TITLE
Wave build timeout

### DISCUSSION THREAD

**Discussion by iferres** [seqeralabs/wave#552] [General]
Hi, I'm using Nextflow `-with-wave`. Is there any possibility the container build is reaching a timeout? The log I get from the services looks good but the status is `FAILED`. The Build ID was [a01237dbf22b56b7_3](https://wave.seqera.io/view/builds/a01237dbf22b56b7_3)

**Comment by pditommaso** [comment]
Hi, currently, 15 minutes is a hard limit. It cannot be modified at the request level.

**Comment by iferres** [comment]
Thank you!
